src/components/ResetPass/index.js

```diff
@@ -0,0 +1,110 @@
+import React, { useState } from "react";
+//Import Material-UI
+import {
+  Box,
+  FormControl,
+  Input,
+  FormHelperText,
+  InputAdornment,
+  IconButton,
+  Card,
+  CardMedia,
+  Typography,
+  CardContent,
+  Button,
+} from "@material-ui/core";
+import { Visibility, VisibilityOff } from "@material-ui/icons";
+//Import Formik
+import { useFormik } from "formik";
+//Import Styles
+import "./style.scss";
+//Import DATA
+import Logo from "assets/images/sushiMochi.jpg";
+
+//Import SchemaValidation
+import Schema from "./schema";
+
+const ResetPass = () => {
+  const email = "TuEmail@dominio.com";
+  const [showNewPass, setShowNewPass] = useState(false);
+  const [showConfirmPass, setShowConfirmPass] = useState(false);
+
+  const formik = useFormik({
+    initialValues: {
+      newPassword: "",
+      confirmPassword: "",
+    },
+    validationSchema: Schema,
+    validateOnChange: false,
+    onSubmit: values => {
```

Comment:
A menos que sea estrictamente necesario, nunca se accede a elementos del DOM directamente en React. Para resetear el form com Formik se hace lo siguiente:

```suggestion
    onSubmit: (values, actions) => {
      actions.resetForm()
    }
    // O con destructuring
    onSubmit: (values, {resetForm}) => {
      resetForm()
    }
```

Reply:
Done!

Reply:
Lo hice con destructuring.